ci/container/test_component.sh

```diff
@@ -71,6 +70,9 @@ python3 $THIS_DIR/hang_webserver.py 12345&
 IFS=','
 read -ra CATEGORY <<< "$JDBC_TEST_CATEGORY" 
 
+# Avoid connection timeout
+export MAVEN_OPTS="-Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=120"
```

Comment:
Should we include old MAVEN_OPTS here? Like `export MAVEN_OPTS="$MAVEN_OPTS -Dhttp...`

Reply:
Indeed we can to make it more extensible in the future